people/martin-popel.md

```diff
@@ -2,7 +2,20 @@
 grand_parent: More
 parent: People
 nav_order: 2
-layout: coming_soon
 title: Martin Popel
-description:
+description: Machine translation researcher
 ---
+
+**Martin Popel** is a machine translation researcher at the [Institute of Formal and Applied Linguistics](https://ufal.mff.cuni.cz/) at Charles University in Prague.
+He is the first-author of CUBBITT, a system for English-Czech translation [published in Nature Communications](https://www.nature.com/articles/s41467-020-18073-9).
+It reaches translation quality comparable to human professionals in the news domain and is available [online](https://lindat.mff.cuni.cz/services/translation/).
+He is the technical editor of [The Prague Bulletin of Mathematical Linguistics](https://ufal.mff.cuni.cz/pbml).
+
+## Research topics
+- mainstream machine translation
```

Comment:
Should we start each list item with a capital letter?

Reply:
If yes, then we need to update ondrej-bojar, lucia-specia and others.

Reply:
No, my bad. It should be in lowercase, as there is no verb in the list introductory sentence. I apologize for the confusion.

However, this made me notice that we haven't followed the [style guide instruction for lists](https://machinetranslate.org/style#lists) in pages such as `lucia-specia.md`, `philipp-koehn.md`, `alon-lavie.md`, and `petr-troyanskii.md`.

> Introduce lists with full sentences.

The `includes:` that introduces lists in the above-mentioned pages forces the sentences to continue in each item, which is not supported in other languages. This can have a negative impact when we machine translate the site.